_posts/2016-05-24-journal-of-a-junior-bem.md

```diff
@@ -0,0 +1,79 @@
+---
+layout:         post
+title:          "Journal of a Junior: BEM Syntax"
+date:           2016-05-25
+author_id:      "maggy_burns"
+description:    "Hello, Maggy here! I’m going to be updating the SB website with journal posts about all the new technologies i’m learning (if I can keep up) and the first post is going to be about BEM syntax. "
+categories:     space between
+banner:         "/assets/images/blog/radcatbem.jpg"
+---
+
+Hello, Maggy here! I’m going to be updating the SB website with journal posts about all the new technologies i’m learning (if I can keep up) and the first post is going to be about BEM syntax. 
+
+BEM - which stands for block, element, modifier - is a front-end naming methodology that the guys at Yandex thought up. I had never heard of BEM syntax before I had started working here at Space Between, and when I first started to look into it, overwhelmed was an understatement! So I dedicated my investment time (which is a beautiful thing)  to it on Friday - and now it makes complete sense and I can’t wait to start using it in future projects. I’m writing this blog post to a) give myself a better understanding and b) hopefully help someone else trying to get their head around it!
+
+###So what exactly is BEM?
+
+BEM is a strict way of arranging your CSS into independent and informative modules. It usually follows this pattern:
+
+<pre><code class="scss">
+.block{}
+.block__element{}
+.block--modifier{}
+
+</code></pre>
+<br>
+A block represent an object on your website, and could be a form, header, or navbar for example.
+A block element represents a descendant of your block and is indicated by beginning with a double underscore ‘__ ‘. Elements are what make up your block, and so can be buttons, textboxes or media items.
+A  modifier is a different version of your element or block and is indicated by double hyphen ‘--’. It could be something like --blue that would give a style like background-color: blue. It’ll indicate that the object would be blue (yes, really!). It can also identify states like booleans, or values like bold, normal or thin for properties like font weight.
```

Comment:
I wouldn't use `--blue` as an example, blocks elements and modifiers should be away from state where possible. Maybe change it to `--active`